### DISCUSSION TITLE
2024/05/24/how-i-got-scammed-out-of-2000-usdt

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#37] [Blog comments]
# 2024/05/24/how-i-got-scammed-out-of-2000-usdt

мои переживания

https://blog.amd-nick.me/2024/05/24/how-i-got-scammed-out-of-2000-usdt

**Comment by exilaurora** [comment]
Не совсем понял. Вы написали, что скопировали адрес с сайта биржи. То есть ваше устройство было взломано? Или все же адрес был скопирован в Trust Wallet?

**Reply by AMD-NICK** [reply]
Скопировал из Trust Wallet. Из фейковой транзакции, которую мне прислали и у которой был очень похожий адрес

**Comment by shockpast** [comment]
Пиздец, грустная ситуация, и ещё грустнее что слышу не впервые про подобное, надеюсь оправишься, желаю всего добра тебе :^)

**Reply by AMD-NICK** [reply]
Те 2000$ я уже похоронил. Сейчас другая проблема в жизни)
